components/datadog/fakeintake/ecsFargate.go

```diff
@@ -34,12 +34,12 @@ func NewECSFargateInstance(e aws.Environment) (*Instance, error) {
 	opts = append(opts, pulumi.Parent(instance))
 
 	alb, err := lb.NewApplicationLoadBalancer(e.Ctx, namer.ResourceName("lb"), &lb.ApplicationLoadBalancerArgs{
```

Comment:
We can add more info to the logical name according to https://www.pulumi.com/docs/concepts/resources/names/, would name this `namer.ResourceName("fakeintake-lb").

Reply:
The logical name already contains `fakeintake` because it’s one of the prefix of the used `namer`: https://github.com/DataDog/test-infra-definitions/blob/18a09ca9510a3c42d887ea73ddcc55bdf1f02006/components/datadog/fakeintake/ecsFargate.go#L26